state/logs.go

```diff
@@ -445,7 +440,7 @@ func (t *logTailer) processReversed(query *mgo.Query) error {
 		return errors.Errorf("too many lines requested (%d) maximum is %d",
 			t.params.InitialLines, maxInitialLines)
 	}
-	query.Sort("-e", "-t", "-_id")
+	query.Sort("-t", "-_id")
```

Comment:
the index we have required e t i, did that change with the split out but we didn't change the query?